Library/ViewModels/ProjectNavBarViewModel.swift

```diff
@@ -37,20 +37,26 @@ public protocol ProjectNavBarViewModelOutputs {
   /// Emits when the login tout should be shown to the user.
   var goToLoginTout: Signal<(), NoError> { get }
 
+  /// Emits a project
+  var project: Signal<Project, NoError> { get }
```

Comment:
I think this could have a better output name since it's not clear what the project is emitting for. How about `postNotificationWithProject`?